include/experimental/group/gemm/impl/int4_dequantize_xe.hpp

```diff
@@ -549,7 +612,12 @@ class gemm_t<
       subgroup::elemwise_cvt(matA_acc, matA);
       dequantize(matB_acc, matB, scale, zero_pt);
       SW_BARRIER();
-      tile_mma::mma(matAcc, matAcc, matB_acc, matA_acc);
+      if constexpr (
+          is_col_major_b && compute_policy::mma_engine == mma_engine::fpu) {
```

Comment:
this mean col major only support fpu engine?
i also notice we add a new fma core, will it only works on next token case?
if true, this mean we should keep 2 weight(row+col major) in vram to handle first & next token?

Reply:
> if true, this mean we should keep 2 weight(row+col major) in vram to handle first & next token?

not possible

Reply:
First token can also support col_major. Continue to use the original xmx or fpu logic. I will give UT later